pkg/types/kustomization.go

```diff
@@ -68,6 +68,12 @@ type Kustomization struct {
 
 	// Variables which will be substituted at runtime
 	Vars []Var `json:"vars,omitempty" yaml:"vars,omitempty"`
+
+	// If set to true, all images need to have tags
+	RequireTag bool `json:"requireTag,omitempty" yaml:"requireTag,omitempty"`
+
+	// Container Reference lists for changing image tags
+	CRs []ContainerRef `json:"cr,omitempty" yaml:"cr,omitempty"`
```

Comment:
`CRs` should be consistent with json|yaml tag `cr`.
Both have `s` or neither has `s`.